#! /bin/sh
patch -p1 -f $* < $0
exit $?

commit 324e61f54
Author: Andrej Kacian <ticho@claws-mail.org>
Date:   Sat Dec 29 14:09:42 2018 +0100

    Emulate strftime's %r on Windows which does not have that

diff --git a/src/common/utils.c b/src/common/utils.c
index feda18d81..c4364c4bd 100644
--- a/src/common/utils.c
+++ b/src/common/utils.c
@@ -4234,7 +4234,11 @@ size_t fast_strftime(gchar *buf, gint buflen, const gchar *format, struct tm *lt
 				}
 				break;
 			case 'r':
+#ifdef G_OS_WIN32
+				strftime(subbuf, 64, "%I:%M:%S %p", lt);
+#else
 				strftime(subbuf, 64, "%r", lt);
+#endif
 				len = strlen(subbuf); CHECK_SIZE();
 				strncpy2(curpos, subbuf, buflen - total_done);
 				break;
